sys/auto_init/netif/auto_init_at86rf2xx.c

```diff
@@ -58,11 +59,19 @@ void auto_init_at86rf2xx(void)
             LOG_ERROR("[auto_init_netif] error initializing at86rf2xx radio #%u\n", i);
         }
         else {
-            gnrc_netdev_init(_at86rf2xx_stacks[i],
-                             AT86RF2XX_MAC_STACKSIZE,
-                             AT86RF2XX_MAC_PRIO,
-                             "at86rf2xx",
-                             &gnrc_adpt[i]);
+#ifdef MODULE_GNRC_LWMAC
+            gnrc_lwmac_init(_at86rf2xx_stacks[i],
+                            AT86RF2XX_MAC_STACKSIZE,
+                            AT86RF2XX_MAC_PRIO,
+                            "at86rf2xx-lwmac",
+                            &gnrc_adpt[i]);
```

Comment:
EM... so, this is the naming problem with multi-radios? (Not sure I get exactly all the point).
So, any suggestion? :-)